modules/Makefile.kubernetes

```diff
@@ -10,9 +10,10 @@ KUBERNETES_RESOURCE_VALIDATION ?= configmap
 export KUBERNETES_APP
 
 # Kubectl specific settings
+SSH_SOCK ?= ~/tmp/kubectl-$(CLUSTER_BASTION)
 KUBECTL ?= /opt/bin/kubectl
 KUBECTL_SCHEMA_CACHE_DIR ?= --schema-cache-dir=.kube/cache
-KUBECTL_SSH_CMD := ssh -A -o 'LogLevel=error' -o 'StrictHostKeyChecking=no' -o 'UserKnownHostsFile=/dev/null'
+KUBECTL_SSH_CMD := ssh -S $(SSH_SOCK)
```

Comment:
I need to change this to add the other options back, since as @jeremymailen points out, these may be getting called without a tunnel in a CI environment